lib/ansible/modules/network/meraki/meraki_admin.py

```diff
@@ -43,6 +43,8 @@
         description:
         - List of networks the administrator has privileges on.
         - When creating a new administrator, C(org_name), C(network), or C(tags) must be specified.
+        - C(id) is the network ID.
```

Comment:
It wasn't clear to me what this new id and access referred to until I looked at the new examples. How about a line in the description that says something like "This list is a json dict with two parameters, C(id) and C(access)."  Also glancing at the code, it seems access can only be set to three choices. Would be worth adding that to this description as well.

Reply:
No, we have a construct for this already. Please use `suboptions`, check any of the modules using, like: *lib/ansible/modules/network/aci/mso_site.py:*